kie-wb-tests/kie-wb-tests-rest/src/main/java/org/kie/wb/test/rest/client/WorkbenchClient.java

```diff
@@ -97,6 +117,21 @@
     CreateOrganizationalUnitRequest createOrganizationalUnit(OrganizationalUnit organizationalUnit);
 
     /**
+     * [POST] /organizationalunits
+     */
+    CreateOrganizationalUnitRequest createOrganizationalUnit(String orgUnitName, String owner);
+
+    /**
+     * [POST] /organizationalunits
+     */
+    CreateOrganizationalUnitRequest createOrganizationalUnit(String orgUnitName, String owner, String description);
+
+    /**
+     * [POST] /organizationalunits
+     */
+    CreateOrganizationalUnitRequest createOrganizationalUnit(String orgUnitName, String owner, String description, String groupId);
```

Comment:
This is fine considering the other parts of the interface. Just a wild thought: would it make sense to have just `CreateOrganizationalUnitRequest createOrganizationalUnit(OrganizationalUnit organizationalUnit)` + create a builder for the `OrganizationalUnit` (which would have optional fields)?